generators/add-action/asset-compute/index.js

```diff
@@ -20,6 +20,9 @@ class AssetComputeGenerator extends ActionGenerator {
   constructor (args, opts) {
     super(args, opts)
     this.props = {}
+    this.option('action-folder', { type: String })
+
+    this.actionFolder = this.options['action-folder']
```

Comment:
this should be already defined in the super class https://github.com/adobe/generator-aio-app/blob/e5fcce7741dd70aa7645ec7d3c434b88e4f40b88/lib/ActionGenerator.js#L29